GANDLF/utils/generic.py

```diff
@@ -114,15 +99,21 @@ def version_check(version_from_config: str, version_to_check: str) -> bool:
     Returns:
         bool: If the version of the config file is compatible with the version of the code.
     """
-    version_to_check_int = parse_version(version_to_check)
-    min_ver = parse_version(version_from_config["minimum"])
-    max_ver = parse_version(version_from_config["maximum"])
-    if (min_ver > version_to_check_int) or (max_ver < version_to_check_int):
-        sys.exit(
-            "Incompatible version of GaNDLF detected ("
-            + str(version_to_check_int)
-            + ")"
-        )
+    version_to_check_int = Version(version_to_check)
+    min_ver = Version(version_from_config.get("minimum"), None)
+    max_ver = Version(version_from_config.get("maximum"), None)
+
+    assert min_ver is not None, "Minimum version is not specified in the config file"
```

Comment:
`min_ver` and `max_ver` cannot be None: they either a valid `Version` object, or execution already failed on a previous line if `version_from_config.get("minimum")` is not a valid version string

Reply:
🤔 I was talking about assert check actually:) it would never be raised as `min_ver` variable is always not a None

Reply:
Ah yes, I understand now. Done.